treeherder/etl/perf.py

```diff
@@ -61,10 +62,14 @@ def _load_perf_datum(job, perf_datum):
         return
     for suite in perf_datum['suites']:
         suite_extra_properties = copy.copy(extra_properties)
+        suite_extra_options = copy.copy(extra_options)
         if suite.get('extraOptions'):
             suite_extra_properties = {
                 'test_options': sorted(suite['extraOptions'])
             }
+            # store extraOptions list as comma-separated str
+            suite_extra_options = ','.join(sorted(suite['extraOptions']))
```

Comment:
Let's use a space as a seperator, rather than a comma. It'll look better in the UI.